packages/remark-language-server/package.json

```diff
@@ -0,0 +1,40 @@
+{
+  "name": "remark-language-server",
+  "version": "1.0.0",
+  "description": "A language server for markdown using remark",
+  "homepage": "https://remark.js.org",
+  "repository": "https://github.com/remarkjs/remark/tree/main/packages/remark-language-server",
+  "bugs": "https://github.com/remarkjs/remark-language-serve/issues",
+  "bin": "remark-language-server.js",
+  "author": "Remco Haszing <remcohaszing@gmail.com>",
+  "license": "MIT",
+  "type": "module",
+  "files": [
+    "remark-language-server.js"
+  ],
+  "scripts": {
+    "format": "remark . -qfo && prettier . -w --loglevel warn && xo --fix",
+    "test": "tsc && npm run format"
```

Comment:
Other packages in this monorepo have different scripts I believe, probably match those?